### PR TITLE
Add more info to the Quote Usage section

### PR DESCRIPTION
**Pull request by gunnaraasen**

### CODE REVIEW

**Comment by toddboom** [comment]
:+1:

File: source/docs/v0.9/query_language/querying_data.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ curl -G 'http://localhost:8086/query' --data-urlencode "q=SELECT * FROM foo"
 ## Quote Usage
 *Identifiers* are either unquoted or double quoted. Identifiers are database names, retention policies, measurements, or tag keys. String literals are always single quoted however.
 
+Double quoted identifiers may contain any unicode character except a new line character. Double quoted identifiers can also contain escaped double quote characters (i.e., `/"`).
```
Comment (dgnorton) [inline source/docs/v0.9/query_language/querying_data.md]: Escaped double quote char should be `\"`.

Comment (toddboom) [inline source/docs/v0.9/query_language/querying_data.md]: Oh, yeah, good catch!

**Comment by gunnaraasen** [comment]
Whoops! Thanks @dgnorton.

**Pull request merged**